qa/run-standalone.sh

```diff
@@ -28,17 +28,17 @@ function cython_module_path() {
             py_ver=3
         fi
     fi
-    echo $ceph_lib/cython_modules/lib.$py_ver
+    echo $(realpath ../src/pybind):$ceph_lib/cython_modules/lib.$py_ver
 }
 
 if [ `uname` = FreeBSD ]; then
     # otherwise module prettytable will not be found
-    export PYTHONPATH=/usr/local/lib/python2.7/site-packages
+    export PYTHONPATH=$(get_python_path):/usr/local/lib/python2.7/site-packages
```

Comment:
@wjwithagen FWIW, i am not trying to address the py3 support on FreeBSD, so i am leaving "python2.7" as it is.

Reply:
@tchaikov 
Oke, that is fine. This could be some trouble in the (near)  future when everything starts to do python3 as preferred. Fortunately this is only executed during manual runs and as such, mostly by devs. And we will fix it there if needed. Finding Prettytable during TOX was already separately fixed in a config.
Will need a working FreeBSD-jenkins before I can test this.